### PR TITLE
Add Intro Cutscene

### PR DESCRIPTION
**Pull request by chuck-stein**
Also organized Animations and Audio folders

### CODE REVIEW

File: Assets/Scripts/IntroCutsceneController.cs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    private void Awake()
+    {
+        Cursor.visible = false;
+        Cursor.lockState = CursorLockMode.Locked;
+    }
+
+    private void Update()
+    {
+        // press escape to skip cutscene
+        if (Input.GetKeyDown(KeyCode.Escape)) LoadFirstLevel();
```
Comment (Unix-Code) [inline Assets/Scripts/IntroCutsceneController.cs]: We might not wanna let them skip the cutscene with esc -> maybe have that as a public field, default it to escape but rewrite on Awake/Start to SPACE or something else if the platform is WebGL (look at the physics cam controller as an example for how to do this).

File: Assets/Scripts/IntroCutsceneController.cs [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        
+        // rotate the skybox to simulate space station moving through space when player looks out the window
+        if (skyboxShouldRotate) RenderSettings.skybox.SetFloat(Rotation, Time.time * spaceStationFlightSpeed);
+    }
+
+    public void StartBemisDialogue()
+    {
+        bemisVoice.Play();
+        Invoke(nameof(EnterSimulationBooth), bemisVoice.clip.length);
```
Comment (Unix-Code) [inline Assets/Scripts/IntroCutsceneController.cs:37]: Rewrite this whole thing as a Coroutine (you can wait for a certain time period or based on a predicate -> ie until bemis audio source isn't playing anymore).

**Review by Unix-Code — CHANGES_REQUESTED** [review]
Some refactor comments

File: Assets/Scripts/IntroCutsceneController.cs (hunk 2 above)
Comment (chuck-stein) [reply Assets/Scripts/IntroCutsceneController.cs:37]: Punting this to post Gold Master: https://chilipot.atlassian.net/browse/PROP-139

**Review by Unix-Code — APPROVED** [review]

File: Assets/Scripts/IntroCutsceneController.cs (hunk 1 above)
Comment (chuck-stein) [reply Assets/Scripts/IntroCutsceneController.cs]: switched to SPACE or RETURN keys

**Pull request merged**